bigtable/admin.go

```diff
@@ -1920,6 +1924,27 @@ func (ac *AdminClient) CreateBackup(ctx context.Context, table, cluster, backup
 	return longrunning.InternalNewOperation(ac.lroClient, op).Wait(ctx, &resp)
 }
 
+// CopyBackup copies the specified source backup with the user-provided expire time.
+func (ac *AdminClient) CopyBackup(ctx context.Context, destProject, destInstance, destCluster, destBackup,
```

Comment:
following the flow of the java client (https://github.com/googleapis/java-bigtable/blob/main/google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/admin/v2/BigtableTableAdminClient.java#L1371-L1373), the order seems to be source parameters, then destination parameters. WDYT, should we reorder or is there a reason to keep as is?

Reply:
Reordered